docs/content/sftp.md

```diff
@@ -187,7 +187,7 @@ SSH host to connect to
 
 #### --sftp-user
 
-SSH username, leave blank for current username, ncw
```

Comment:
Ha!

We set USER in the makefile to try to avoid this but it obviously isn't working!